app/src/main/java/org/sil/storyproducer/model/Phase.kt

```diff
@@ -3,146 +3,169 @@ package org.sil.storyproducer.model
             PhaseType.REMOTE_CHECK -> R.drawable.ic_school_white_48dp
-            PhaseType.BACKT -> R.drawable.ic_headset_mic_white_48dp
-            PhaseType.DRAMATIZATION -> R.drawable.ic_mic_box_48dp
+            PhaseType.BACK_TRANSLATION -> R.drawable.ic_headset_mic_white_48dp
+            PhaseType.VOICE_STUDIO -> R.drawable.ic_mic_box_48dp
             else -> R.drawable.ic_mic_white_48dp
         }
     }
 
+    /**
+     * get the color associated with a phase
+     * @return color
+     */
+    fun getColor() : Int {
+        return when(phaseType){
+            PhaseType.LEARN -> R.color.learn_phase
+            PhaseType.TRANSLATE_REVISE -> R.color.translate_revise_phase
+            PhaseType.COMMUNITY_WORK -> R.color.comunity_work_phase
+            PhaseType.ACCURACY_CHECK -> R.color.accuracy_check_phase
+            PhaseType.VOICE_STUDIO -> R.color.voice_studio_phase
+            PhaseType.FINALIZE -> R.color.finalize_phase
+            PhaseType.SHARE -> R.color.share_phase
+            PhaseType.BACK_TRANSLATION -> R.color.backT_phase
+            PhaseType.WHOLE_STORY -> R.color.whole_story_phase
+            PhaseType.REMOTE_CHECK -> R.color.remote_check_phase
+            else -> R.color.black
+        }
+    }
+
+    /**
+     * get path of audio file needed by slide, based on the current phase (narration or a draft)
+     * @return String
+     */
     fun getReferenceAudioFile(slideNum: Int = Workspace.activeSlideNum) : String {
         val filename = when (phaseType){
-            PhaseType.DRAFT -> Workspace.activeStory.slides[slideNum].narrationFile
-            PhaseType.COMMUNITY_CHECK -> Workspace.activeStory.slides[slideNum].chosenDraftFile
-            PhaseType.CONSULTANT_CHECK -> Workspace.activeStory.slides[slideNum].chosenDraftFile
-            PhaseType.DRAMATIZATION -> Workspace.activeStory.slides[slideNum].chosenDraftFile
-            PhaseType.BACKT -> Workspace.activeStory.slides[slideNum].chosenDraftFile
+            PhaseType.TRANSLATE_REVISE -> Workspace.activeStory.slides[slideNum].narrationFile
+            PhaseType.COMMUNITY_WORK -> Workspace.activeStory.slides[slideNum].chosenDraftFile
+            PhaseType.ACCURACY_CHECK -> Workspace.activeStory.slides[slideNum].chosenDraftFile
+            PhaseType.VOICE_STUDIO -> Workspace.activeStory.slides[slideNum].chosenDraftFile
+            PhaseType.BACK_TRANSLATION -> Workspace.activeStory.slides[slideNum].chosenDraftFile
             else -> ""
         }
         return Story.getFilename(filename)
     }
 
-    fun getPrettyName() : String {
+    /**
+     * get displayable name for a phase (used to create option menu help dialog and logging)
+     * @return String
+     */
+    fun getDisplayName() : String {
         return when (phaseType) {
             PhaseType.LEARN -> "Learn"
-            PhaseType.DRAFT -> "Translate"
-            PhaseType.CREATE -> "Finalize"
+            PhaseType.TRANSLATE_REVISE -> "Translate + Revise"

```

Comment:
Display names, since they are used in the UI, should be defined in the string resource file, not here in the code.
I wrote issue #546 to do this later. A good task for Dale!